docs/index.rst

```diff
@@ -149,7 +149,7 @@ django CMS   Django                                      Python
 3.2.1-3.2.5  ⨯          ✓          ✓    ✓    ⨯     ⨯     ✓    ✓    ✓    ✓    ✓    ⨯
 3.3.0-3.4.1  ⨯          ⨯          ✓    ✓    ⨯     ⨯     ⨯    ✓    ✓    ✓    ✓    ⨯
 3.4.2-3.4.4  ⨯          ⨯          ✓    ✓    ✓     ⨯     ⨯    ✓    ✓    ✓    ✓    ⨯
-3.4.5        ⨯          ⨯          ✓    ✓    ✓     ✓     ⨯    ✓    ✓    ✓    ✓    ✓
+3.4.5-3.5.0  ⨯          ⨯          ✓    ✓    ✓     ✓     ⨯    ✓    ✓    ✓    ✓    ✓
```

Comment:
We won't change Django compatibility so I suggest to use 3.5.x to avoid having to update this.